public/electron/fuse/askpass.osascript.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env osascript -l JavaScript
```

Comment:
what is the purpose this script? I understand that you are asking for the user password, but what is used for?

Reply:
Yes, the script is to collect the user's password, this is required by the installer to install the kernel extension. [See here](https://github.com/FleekHQ/space-desktop/pull/357/files#diff-3f7c7709e21da870873cf2f8b33600761bd27a83517c5169ff7c690a57017eb6R49) where I pass the script to the installer process